webscraping/r/aeroeng_document.json

```diff
@@ -0,0 +1,181 @@
+        "note": "Students who have received credit for MECH 485 or for this topic under a MECH 498 number may not take this course for credit."
+    },
+    {
+        "code": "AERO 486",
+        "name": "Aircraft Stress Analysis",
+        "credits": "3",
+        "prereq.string": "ENGR 243, 244",
+        "description": "Definition of load paths in typical aircraft structures. Derivation of analysis procedures to enable the designer to size preliminary designs. Internal shear flow distributions that balance external loads. Stress analysis of open and closed cell beams; statically indeterminate beams and frames; single and multi cell torque boxes; symmetric heavy fuselage frames. Structural instability of columns, beams, plates and flanges in compression and shear. Centres of twist and flexure; structural warping; margins of safety; concepts of optimum design; lug analysis and mechanical joints; matrix analysis methods leading to the Finite Element method. Stress analysis of thin-walled metallic structures. ",
+        "lecture.hours": "three hours per week",
+        "note": "Students who have received credit for MECH 486 may not take this course for credit."
+    },
+    {
+        "code": "AERO 487",
+        "name": "Design of Aircraft Structures",
+        "credits": "3",
+        "prereq.string": "AERO 486",
+        "description": "Design process for aircraft structures. Aero/performance aspects of aircraft structures. Airworthiness and design considerations. Materials. Static, vibratory and aeroelastic loadings. Propulsion-induced loadings. Functions and fabrication of structural components. Design for buckling of aircraft structures: local buckling, instability of stiffened panels, flexural torsional buckling. Design for fracture and fatigue failures. Stress analysis and design of wings, fuselages, stringers, fuselage frames, wing ribs, cut-outs in wings and fuselages, and laminated structures. Design using Finite Element Method. Concept of Optimum Design of Aircraft Structures. Design case studies. ",
+        "lecture.hours": "three hours per week",
+        "note": "Students who have received credit for MECH 487 may not take this course for credit."
+    },
+    {
+        "code": "AERO 490",
+        "name": "Capstone Aerospace Engineering Design Project",
+        "credits": "4",
+        "prereq.string": "75 credits in the program; AERO 390; ENCS 282; ENGR 301",
+        "description": "A supervised design, simulation or experimental capstone design project including a preliminary project proposal with complete project plan and a technical report at the end of the fall term; a final report by the group and presentation at the end of the winter term. ",
+        "lecture.hours": "one hour per week, one term",
+        "note": "Students will work in groups under direct supervision of a faculty member.NOTE: Students who have received credit for COEN 490 or ELEC 490 or MECH 490 may not take this course for credit."
+    }
+]
+
```

Comment:
Why do we need both aeroeng_document.json _and/or_  aeroeng_full-course-info.json?
Also is there only aeroeng course data because you only ran one test **for now** and saved it?
And can we test this on other programs outside of engineering/CS?